apps/condo/domains/billing/components/BillingPageContent/PaymentsTab.tsx

```diff
@@ -0,0 +1,52 @@
+import get from 'lodash/get'
+import React, { CSSProperties } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+import { Typography } from '@open-condo/ui'
+
+import PaymentsTable from '@condo/domains/acquiring/components/payments/PaymentsTable'
+import { PAYMENT_DONE_STATUS, PAYMENT_WITHDRAWN_STATUS } from '@condo/domains/acquiring/constants/payment'
+import { Payment } from '@condo/domains/acquiring/utils/clientSchema'
+import { BasicEmptyListView } from '@condo/domains/common/components/EmptyListView'
+import { Loader } from '@condo/domains/common/components/Loader'
+
+const SEARCHING_DINO_IMG = 'dino/searching@2x.png'
+const IMG_STYLES: CSSProperties = { marginBottom: 24 }
+const TEXT_GAP = 16
+
+
+export const PaymentsTab: React.FC = () => {
+    const intl = useIntl()
+    const NoPaymentsTitle = intl.formatMessage({ id:'accrualsAndPayments.payments.noPaymentsYet.title' })
+    const NoPaymentsMessage = intl.formatMessage({ id:'accrualsAndPayments.payments.noPaymentsYet.message' })
+
+    const { organization } = useOrganization()
+    const orgId = get(organization, 'id', null)
+    const { objs: anyPayments, loading: anyPaymentsLoading, error: anyPaymentsError } = Payment.useObjects({
```

Comment:
Why not `useObject`?

Reply:
I think it's a matter of taste, since they're all using basic `useQuery` under the hood. For me `useObjects` is more break-safe for future refactoring.

If you have any points for `useObject` usage over `useObjects` I'll change that with no problem :)

Reply:
It's all about readability. If you just need to receive one object IMO you should use `useObject`. Otherwise you can use `useObjects`. There are no any other points, so you're correct

Reply:
I need to receive not 1 object, but "at least" one object. First is just a limitation to probably cache-hit better